File: Java/media/src/main/java/ar/org/dracul/dto/FilePaginateDTO.java
package ar.org.dracul.dto;

import java.util.List;

public class FilePaginateDTO {

    private List<FileDTO> items;
    private Integer totalItems;
    private Integer page;

    public List<FileDTO> getItems() {
        return items;
    }

    public void setItems(List<FileDTO> items) {
        this.items = items;
    }

    public Integer getTotalItems() {
        return totalItems;
    }

    public void setTotalItems(Integer totalItems) {
        this.totalItems = totalItems;
    }

    public Integer getPage() {
        return page;
    }

    public void setPage(Integer page) {
        this.page = page;
    }
    
    public Boolean exists() {		
		return this.totalItems > 0;
	}


}

File: PHP/FilePaginateDTO.php
<?php

class FilePaginateDTO {

    public array $items;
    public int $totalItems;
    public int $page;

    public function getItems() {
        return $this->items;
    }

    public function setItems($items) {
        $this->items = $items;
    }

    public function getTotalItems() {
        return $this->totalItems;
    }

    public function setTotalItems($totalItems) {
        $this->totalItems = $totalItems;
    }

    public function getPage() {
        return $this->page;
    }

    public function setPage($page) {
        $this->page = $page;
    }
    
    public function exists() {		
		return $this->totalItems > 0;
	}


}
